clientconn.go

```diff
@@ -1223,6 +1260,11 @@ func (ac *addrConn) updateConnectivityState(s connectivity.State, lastErr error)
 	if ac.state == s {
 		return
 	}
+	locality, backendService := fetchLabels(ac)
+	if ac.state == connectivity.Ready || (ac.state == connectivity.Connecting && s == connectivity.Idle) {
+		disconnectionsMetric.Record(ac.cc.metricsRecorderList, 1, ac.cc.target, backendService, locality, "unknown")
```

Comment:
Why is the `disconnect_error` label set to unknown here? The gRFC talks about a set of possible values for this label.